consensus-types/light-client/bootstrap.go

```diff
@@ -206,3 +216,71 @@ func (h *bootstrapDeneb) CurrentSyncCommittee() *pb.SyncCommittee {
 func (h *bootstrapDeneb) CurrentSyncCommitteeBranch() interfaces.LightClientSyncCommitteeBranch {
 	return h.currentSyncCommitteeBranch
 }
+
+func (h *bootstrapDeneb) CurrentSyncCommitteeBranchElectra() interfaces.LightClientSyncCommitteeBranchElectra {
+	return [6][32]byte{}
+}
+
+type bootstrapElectra struct {
+	p                          *pb.LightClientBootstrapElectra
+	header                     interfaces.LightClientHeader
+	currentSyncCommitteeBranch interfaces.LightClientSyncCommitteeBranchElectra
+}
+
+var _ interfaces.LightClientBootstrap = &bootstrapElectra{}
+
+func NewWrappedBootstrapElectra(p *pb.LightClientBootstrapElectra) (interfaces.LightClientBootstrap, error) {
+	if p == nil {
+		return nil, consensustypes.ErrNilObjectWrapped
+	}
+	header, err := NewWrappedHeaderElectra(p.Header)
+	if err != nil {
+		return nil, err
+	}
+	branch, err := createBranch[interfaces.LightClientSyncCommitteeBranchElectra](
+		"sync committee",
+		p.CurrentSyncCommitteeBranch,
+		fieldparams.SyncCommitteeBranchDepth,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return &bootstrapElectra{
+		p:                          p,
+		header:                     header,
+		currentSyncCommitteeBranch: branch,
+	}, nil
+}
+
+func (h *bootstrapElectra) MarshalSSZTo(dst []byte) ([]byte, error) {
+	return h.p.MarshalSSZTo(dst)
+}
+
+func (h *bootstrapElectra) MarshalSSZ() ([]byte, error) {
+	return h.p.MarshalSSZ()
+}
+
+func (h *bootstrapElectra) SizeSSZ() int {
+	return h.p.SizeSSZ()
+}
+
+func (h *bootstrapElectra) Version() int {
+	return version.Electra
+}
+
+func (h *bootstrapElectra) Header() interfaces.LightClientHeader {
+	return h.header
+}
+
+func (h *bootstrapElectra) CurrentSyncCommittee() *pb.SyncCommittee {
+	return h.p.CurrentSyncCommittee
+}
+
+func (h *bootstrapElectra) CurrentSyncCommitteeBranch() interfaces.LightClientSyncCommitteeBranch {
+	return [5][32]byte{}
```

Comment:
Same here, I think there should be a better way to deal with this. Open to suggestions

Reply:
This should be an error in Electra. You should change the signature of this function to return `(interfaces.LightClientSyncCommitteeBranch, error)`.